### PR TITLE
Suggest alternatives for FtpWebRequest

### PR DESCRIPTION
**Pull request by caesar-chen**
Related: https://github.com/dotnet/corefx/issues/23404

/cc: @davidsh @karelz

### CODE REVIEW

File: docs/DE0003.md [hunk 1]
```diff
@@ -14,4 +14,5 @@ T:System.Net.HttpWebRequest
 
 ## Recommendation
 
-Use `HttpClient` instead.
+For HttpWebRequest: use `HttpClient` instead.
+For FtpWebRequest: search for thrid party FTP client.
```
Comment (davidsh) [inline docs/DE0003.md]: spelling: "thrid" -> "third"

**Comment by karelz** [comment]
@geoffkizer @stephentoub I think one of you mentioned some specific popular FTP client library during 2.0 time, do you remember?
We could link https://stackoverflow.com/questions/1371964/free-ftp-library for some examples.

File: docs/DE0003.md [hunk 2]
```diff
@@ -14,4 +14,5 @@ T:System.Net.HttpWebRequest
 
 ## Recommendation
 
-Use `HttpClient` instead.
+For HttpWebRequest: use `HttpClient` instead.
+For FtpWebRequest: search for third party FTP client.
```
Comment (halter73) [inline docs/DE0003.md:18]: Recommending a specific FTP client library that we know targets netstandard would be really nice.

Comment (karelz) [reply docs/DE0003.md:18]: @halter73 do you have a specific recommendation beside the SO link above?

Comment (halter73) [reply docs/DE0003.md:18]: Nope. Sorry. That's why I was hoping for one.

**Pull request closed without merging**